composer.json

```diff
@@ -15,6 +15,6 @@
   },
   "conflict": {
     "drupal/core": "<=8",
-    "islandora/islandora_defaults": "<=2.1.1"
+    "islandora/islandora_defaults": "<=3"
```

Comment:
Shouldn't this be < 3 rather than <= 3, since we do want 3.0.0 to be compatible?

Reply:
Ah, yeah, whoops. Was wondering why I was ending up with the `3.x-dev` version, and that could explain it (with `3` referring to `3.0.0`).

Addressed in https://github.com/Islandora/islandora_mirador/pull/8/commits/77446fbc8918757e138bc975745fb71f776dd69c